AirCasting/Utils/CLLocationUtils.swift

```diff
@@ -0,0 +1,35 @@
+// Created by Lunar on 14/06/2021.
+//
+
+import CoreLocation
+
+extension CLLocationCoordinate2D: Equatable { }
+
+extension CLLocationCoordinate2D {
+    public init?(latitude: Double?, longitude: Double?) {
```

Comment:
I would be glad You wouldn't.. The check in places where coordinates is mapped to CLLocationCoordinate2D it is lots more understandable that it can fail.

Reply:
Not too convinced since lat/long can be `null` in our API responses it's quite handy to have a failable initializer for that:

```Swift
DatabaseSession(
...
location: CLLocationCoordinate2D(latitude: $0.latitude, longitude: $0.longitude),
...
)
```

But I'm not too attached to that so let's change 👌